agb-image-converter/src/lib.rs

```diff
@@ -115,15 +115,12 @@ pub fn include_aseprite_inner(input: TokenStream) -> TokenStream {
 
         for frame in frames {
             let width = frame.width();
-            assert!(width == frame.height() && width.is_power_of_two() && width <= 64);
+            let height = frame.height();
+            assert!(width.is_power_of_two() && width <= 64);
+            assert!(height.is_power_of_two() && height <= 64);
```

Comment:
Do you want to handle the invalid sizes at compile time rather than runtime?

Reply:
Technically it's still at compile time just later on (gets evaluated in a const context). But we could check for invalid ones at this point, I didn't like the idea of duplicating code too much, but this part of the code is not likely to change (don't think we'll be seeing an upgrade to the gba to support more sprite sizes).